src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -2,13 +2,44 @@
 
 import core.basesyntax.dao.StorageDao;
 import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.exception.RegistrationException;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int minAge = 18;
+    private static final int minCredentialLength = 6;
```

Comment:
The constant 'minCredentialLength' should follow Java constant naming conventions with all uppercase letters and underscores: MIN_CREDENTIAL_LENGTH.